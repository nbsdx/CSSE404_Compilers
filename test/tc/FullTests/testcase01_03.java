class A1 {
    public static void main(String[] a2) {
	System.out.println(1);
    }
}

class double {}

class A3 {
    public boolean a6(boolean a7) {
	return !a7;
    }
}

class A8 extends A3 {
    public boolean a9(boolean a7) {
	if(a7 && 3) { //Int instead of boolean
	} else {}
	return true;
    }
}
