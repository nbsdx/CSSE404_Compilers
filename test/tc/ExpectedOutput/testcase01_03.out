Right argument of type int does not match expected type boolean for operator AND
